tensorflow/python/keras/benchmarks/keras_examples_benchmarks/README.md

```diff
@@ -0,0 +1,183 @@
+| :---: | :--------: | :-------: | :------------: | :---------: | :-------------------: |
+|  CPU  |    256     |  654.05   |     218.68     |   274.24    |         `off`         |
+| GPU:2 |    256     |   20.77   |      3.73      |  15088.06   |      `mirrored`       |
+
+#### Bidirectional LSTM benchmark
+
+|       | Batch_size | Wall_time | Avg_epoch_time | Exp_per_sec | Distribution_Strategy |
+| :---: | :--------: | :-------: | :------------: | :---------: | :-------------------: |
+|  CPU  |    512     |  225.57   |     72.55      |   344.70    |         `off`         |
+| GPU:2 |    512     |   23.54   |      3.23      |   7532.53   |      `mirrored`       |
+
+#### Text classification with transformer benchmark
+
+|       | Batch_size | Wall_time | Avg_epoch_time | Exp_per_sec | Distribution_Strategy |
+| :---: | :--------: | :-------: | :------------: | :---------: | :-------------------: |
+|  CPU  |    512     |  109.22   |     35.93      |   698.10    |         `off`         |
+| GPU:2 |    512     |   9.28    |      0.83      |  26567.54   |      `mirrored`       |
+
+#### MLP benchmark
+
+|       | Batch_size | Wall_time | Avg_epoch_time | Exp_per_sec | Distribution_Strategy |
+| :---: | :--------: | :-------: | :------------: | :---------: | :-------------------: |
+|  CPU  |    128     |   3.76    |      0.54      |  17678.54   |         `off`         |
+| GPU:2 |    128     |   5.91    |      0.30      |  25435.14   |      `mirrored`       |
+
+#### Antirectifier benchmark
+
+|       | Batch_size | Wall_time | Avg_epoch_time | Exp_per_sec | Distribution_Strategy |
+| :---: | :--------: | :-------: | :------------: | :---------: | :-------------------: |
+|  CPU  |    512     |   6.77    |      1.79      |  30916.39   |         `off`         |
+| GPU:2 |    512     |   6.81    |      0.66      |  66563.17   |      `mirrored`       |
+
+#### IRNN benchmark
+
+|       | Batch_size | Wall_time | Avg_epoch_time | Exp_per_sec | Distribution_Strategy |
+| :---: | :--------: | :-------: | :------------: | :---------: | :-------------------: |
+|  CPU  |    1024    |  213.00   |     69.01      |   868.08    |         `off`         |
+| GPU:2 |    1024    |   92.71   |     29.12      |   2042.94   |      `mirrored`       |
+
+**Note**: For the smaller models, running models with GPU may be slower than running models with CPU as training small models is not computation dominant and there might be some overhead on model replication and data sharding with distributed training on GPUs.
+
+## Installing Bazel
+
+This step can be skipped if Bazel is already installed. </br>
+
+We need to use [Bazel](https://bazel.build/) to build targets based on BUILD files. It will take a while for the first time because it will compile all dependencies from your BUILD file. For the next time, Bazel will use the cache and it’ll be much faster. Since we use Ubuntu OS, we can install bazel by using apt repository.
```

Comment:
"We need to use [Bazel](https://bazel.build/)....." ---> "[Bazel](https://bazel.build/) is used ....."

"Since we use Ubuntu OS, we can install bazel by using apt repository." -- > "For Ubuntu OS, please use the following steps for Bazel installation. For other platforms, you may follow the corresponding guide for the installation."